### PR TITLE
[#6412] Bump version number to 4.3.1 (4-3-stable)

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -84,9 +84,9 @@ include(IrodsRunpathDefaults)
 
 set(IRODS_VERSION_MAJOR "4")
 set(IRODS_VERSION_MINOR "3")
-set(IRODS_VERSION_PATCH "0")
+set(IRODS_VERSION_PATCH "1")
 set(IRODS_VERSION "${IRODS_VERSION_MAJOR}.${IRODS_VERSION_MINOR}.${IRODS_VERSION_PATCH}")
-set(IRODS_PACKAGE_REVISION "1")
+set(IRODS_PACKAGE_REVISION "0")
```
Comment (trel) [inline CMakeLists.txt:89]: i'm not sure about this setting to 0...

**Review by trel — COMMENTED** [review]
and i think need the Doxyfile as well...

https://github.com/korydraughn/irods/commit/1daecb965f7df72f0ffd649e93540433f34f9ef1

File: CMakeLists.txt (hunk 1 above)
Comment (korydraughn) [reply CMakeLists.txt:89]: See https://github.com/irods/irods/issues/6412#issuecomment-1724656181.

I was unsure about changing that too, but that comment seems to imply this repo's package revision should be updated too.

@SwooshyCueb Should the package revision number for this repo be changed to 0 or left as 1?

Comment (SwooshyCueb) [reply CMakeLists.txt:89]: Should be changed to 0

Comment (trel) [reply CMakeLists.txt:89]: He has spoken.

**Comment by korydraughn** [comment]
Updated Doxyfile.

**Comment by trel** [comment]
Good.   Please squash.

**Comment by korydraughn** [comment]
Squashed.

**Review by trel — APPROVED** [review]
please to pound

**Comment by korydraughn** [comment]
#'d

**Review by alanking — APPROVED** [review]

**Comment by SwooshyCueb** [comment]
Please cherry-pick this into main

**Pull request merged**